src/components/Header/Header.jsx

```diff
@@ -73,6 +81,64 @@ const ToolbarHeader = styled(Toolbar)`
   }
 `;
 
+const LinkGroup = ({ navbarPages }) => {
+  const router = useReactRouter();
```

Comment:
I believe this is the first React Hook we're using. I don't have a ton of experience with them, do they provide a significant benefit to developer experience?

Reply:
Hooks are quite convenient to use & are also awesome to be used with type analysis tools like typescript/flow